rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -76,13 +75,17 @@
     public static final String PROPERTY_PEER_ACTIVE = "peer.active";
     public static final String PROPERTY_DB_RESET = "database.reset";
     // TODO review rpc properties
-    public static final String PROPERTY_RPC_ENABLED = "rpc.enabled";
-    public static final String PROPERTY_RPC_PORT = "rpc.port";
-    public static final String PROPERTY_RPC_HOST = "rpc.host";
-    public static final String PROPERTY_RPC_CORS = "rpc.cors";
-    public static final String PROPERTY_RPC_ADDRESS = "rpc.address";
+    public static final String PROPERTY_RPC_CORS = "rpc.providers.web.cors";
+    public static final String PROPERTY_RPC_HTTP_ENABLED = "rpc.providers.web.http.enabled";
+    public static final String PROPERTY_RPC_HTTP_ADDRESS = "rpc.providers.web.http.bind_address";
+    public static final String PROPERTY_RPC_HTTP_HOSTS = "rpc.providers.web.http.hosts";
+    public static final String PROPERTY_RPC_HTTP_PORT = "rpc.providers.web.http.port";
+    private static final String PROPERTY_RPC_WEBSOCKET_ENABLED = "rpc.providers.web.ws.enabled";
+    private static final String PROPERTY_RPC_WEBSOCKET_ADDRESS = "rpc.providers.web.ws.bind_address";
+    private static final String PROPERTY_RPC_WEBSOCKET_PORT = "rpc.providers.web.ws.port";
+
     public static final String PROPERTY_PUBLIC_IP = "public.ip";
-    public static final String PROPERTY_BIND_ADDRESS = "bind.address";
```

Comment:
`SystemProperties.java` uses `configFromFiles.hasPath(PROPERTY_BIND_ADDRESS) do you have to add the `rpc.providers.web.http` prefix?

Reply:
No, this is the `bind_address` of the _peer server_ 🙈